src/Components/RecommendationRating/RecommendationRating.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { cleanup, render } from '@testing-library/react';
+import RecommendationRating from './RecommendationRating';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+it ('matches snapshot with positive feedback', ()=>{
```

Comment:
It's better to wrap test cases in `describe` for instance these could be wrapper in `RecommendationRating`. It's not a mistake, but it wraps a few tests in one nice group so you can see rather quickly what failed and which component you need to look at.

Reply:
Thank you @karelhala for review. Added test cases inside `describe` and also tried to add query selectors inside snapshot. 
Please have a look at new changes & suggest if any additional changes required.